lib/cli/commands/generate.js

```diff
@@ -2,7 +2,48 @@ import fs from 'fs';
 import path from 'path';
 import ui from '../lib/ui';
 import Command from '../lib/command';
+import discoverAddons from '../../utils/discover-addons';
+import topsort from '../../utils/topsort';
 import padEnd from 'lodash/padEnd';
+import find from 'lodash/find';
+
+// TODO merge this with lib/runtime/addon.js
+class Addon {
```

Comment:
Not sure how I could merge this with the `runtime/addon` since that needs things like the `container` to init,
and the generate command doesn't have access to that.